substrate/frame/identity/src/lib.rs

```diff
@@ -166,7 +187,7 @@ pub mod pallet {
 		_,
 		Twox64Concat,
 		T::AccountId,
-		Registration<BalanceOf<T>, T::MaxRegistrars, T::IdentityInformation>,
+		(Registration<BalanceOf<T>, T::MaxRegistrars, T::IdentityInformation>, Option<Username>),
```

Comment:
Any special reason for not including the `Option<Username>` within `Registration`? from my pov, it seems like a cleaner approach.

Reply:
The `Registration` only contains information associated with a registrar. It makes sense to keep the username separate, but without the need to introduce another storage map.